tools/inspect/deprecated_include_check.cpp

```diff
@@ -29,7 +29,9 @@ namespace boost
       { "boost/unordered_set\\.hpp", "unordered_set" },
       { "boost/utility/enable_if\\.hpp", "type_traits" },
       { "boost/detail/atomic_count\\.hpp", "hpx/util/atomic_count.hpp" },
-      { "boost/function\\.hpp", "hpx/util/function.hpp" },
```

Comment:
I didn't even noticed I had removed it, and later added a duplicated check. Will rewrite the fix.